index.js

```diff
@@ -0,0 +1,60 @@
+let opcao_select_diamante = document.getElementById('select-diamante');
```

Comment:
```suggestion
let opcaoSelectDiamante = document.getElementById('select-diamante');
```

Manter o padrão de camelCase